src/chainstate/stacks/boot/pox-2.clar

```diff
@@ -63,8 +64,6 @@
 (define-map stacking-state
     { stacker: principal }
     {
-        ;; how many uSTX locked?
-        amount-ustx: uint,
```

Comment:
Do you think you could provide a read-only function that returns the number of locked tokens for a principal, via `stx-account`?  Asking because I'm pretty sure stacking.club depends on this information (and there could be others).